modules/monitoring-configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts_{context}"]
+= Configuring different alert receivers for default platform alerts and user-defined alerts
+
+You can configure different alert receivers for default platform alerts and user-defined alerts to ensure the following results:
+
+* All default platform alerts are sent to a receiver owned by the team in charge of these alerts.
+* All user-defined alerts are sent to another receiver so that the team can focus only on platform alerts.
+
+You can achieve this by using the `openshift_io_alert_source="platform"` label that is added by the Cluster Monitoring Operator to all platform alerts.
+
+.Prerequisites
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
+endif::openshift-dedicated,openshift-rosa[]
+
+.Procedure
+
+To configure different alert receivers from the CLI:
+
+. Print the Alertmanager configuration into the `alertmanager.yaml` file:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring get secret alertmanager-main --template='{{ index .data "alertmanager.yaml" }}' | base64 --decode > alertmanager.yaml
+----
++
+. Edit the `alertmanager.yaml` file:
++
+[source,yaml]
+----
+...
+route:
+  routes:
+    - matchers:
+        - '"openshift_io_alert_source=platform"'
+      receiver: <defaul-platform-alert-receiver> # <1>
+    - matchers:
+        - '"openshift_io_alert_source!=platform"'
+      receiver: <user-defined-alert-receiver> # <2>
+...
+----
+<1> Replace `<defaul-platform-alert-receiver>` with the receiver where you want to send all the default platform alerts.
+<2> Replace `<user-defined-alert-receiver>` with the receiver where you want to send all the user-defined alerts.
+
```

Comment:
NOTE: receivers.name should be changed correspondingly, example
```
inhibit_rules:
  - equal:
      - namespace
      - alertname
    source_matchers:
      - severity = critical
    target_matchers:
      - severity =~ warning|info
  - equal:
      - namespace
      - alertname
    source_matchers:
      - severity = warning
    target_matchers:
      - severity = info
receivers:
  - name: Default
  - name: platform_alert
    email_configs:
      - to: *****
        from: *****
        smarthost: '*****:25'
        auth_username: *****
        auth_password: *****
        require_tls: false
  - name: user_alert
    email_configs:
      - to: *****
        from: *****
        smarthost: '*****:25'
        auth_username: *****
        auth_password: *****
        require_tls: false
route:
  group_by:
    - namespace
  group_interval: 5m
  group_wait: 30s
  receiver: Default
  repeat_interval: 12h
  routes:
    - matchers:
        - 'openshift_io_alert_source=platform'
      receiver: platform_alert 
    - matchers:
        - 'openshift_io_alert_source!=platform'
      receiver: user_alert
```

Reply:
Hello @juzhao I assumed the receivers would already be created and the user would use the name of an existing receiver. Maybe I should clarify it in the callout better? Something like:

Replace `<defaul-platform-alert-receiver>` with the name of the receiver where you want to send all the default platform alerts.

Is that what you meant?

Reply:
yes, that's more clearly. @eromanova97